airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresSource.java

```diff
@@ -464,6 +471,25 @@ public static void main(final String[] args) throws Exception {
     LOGGER.info("completed source: {}", PostgresSource.class);
   }
 
+  @Override
+  public AirbyteConnectionStatus check(final JsonNode config) throws Exception {
+    if (isCdcReplicationMethod(config)){
+      String sslModeValue = config.get(SSL_MODE).get(MODE).asText();
+      if(SSL_MODE_ALLOW.equals(sslModeValue) || SSL_MODE_PREFER.equals(sslModeValue) ){
```

Comment:
nitpick: slightly nicer to define a `private static final Set<String> INVALID_CDC_SSL_MODES = ImmutableSet.of(SSL_MODE_ALLOW, SSL_MODE_PREFER)` and then do `if (INVALID_CDC_SSL_MODES.conatins(sslModeValue))` here

or maybe define `VALID_CDC_SSL_MODES` and use `if (!VALID_CDC_SSL_MODES.contains(..))`, idk